manage.sh

```diff
@@ -432,14 +434,72 @@ events_file_env(){
 	return 0
 }
 
+# Function that updates bitcoin.conf and Config.toml
+update_configs(){
+    if [ "${NETWORK}" == "testnet" ]; then
+		BTC_HOST=${TBTC_HOST}
+		BTC_RPC_USER=${TBTC_RPC_USER}
+		BTC_RPC_PASS=${TBTC_RPC_PASS}
+		BTC_RPC_PORT=${TBTC_RPC_PORT}
+		BTC_P2P_PORT=${TBTC_P2P_PORT}
+    fi
+	CONFIG_TOML="${SCRIPTPATH}/conf/${NETWORK}/Config.toml"
+	BTC_CONF="${SCRIPTPATH}/conf/${NETWORK}/bitcoin.conf"
+	${VERBOSE} && log "update_configs BTC_HOST: ${BTC_HOST}"
```

Comment:
Probably don't need to debug log these variable values

Reply:
This part isn't my code, it was probably added just to debug.
Ok, commented out.